woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php

```diff
@@ -538,6 +538,81 @@ private function read( \WC_Payment_Token $core_token ) {
 	}
 
 
+	/**
+	 * Stores the token data in the database.
+	 *
+	 * Stores the token data as a Woocommerce payment token, in the `wp_woocommerce_payment_tokens` table,
+	 * with meta data in the `wp_woocommerce_payment_tokenmeta` table.
+	 *
+	 * @since 5.6.0-dev.1
+	 */
+	public function save() {
+
+		if ( $this->token instanceof \WC_Payment_Token ) {
+			$token = $this->token;
+		} else {
+
+			// see if there is already a token with this ID saved for this customer and gateway
```

Comment:
I was thinking maybe this part (the variable assignment and the foreach) could be moved to `get_woocommerce_payment_token`

Reply:
yeah actually I was thinking that too, because there's a couple of instances (save, delete) where we would need this - sounds good to me